py_gtfs_rt_ingestion/tests/test_configuration.py

```diff
@@ -2,37 +2,41 @@
 
 from datetime import datetime
 from py_gtfs_rt_ingestion import Configuration
+from py_gtfs_rt_ingestion import ConfigType
 
 def test_filname_parsing():
     """
     Check that we are able to get the correct Configuration type for multiple
     filenames
     """
-    trip_updates_type = Configuration.from_filename(
+    trip_updates_type = Configuration(
```

Comment:
Update test to work with re-factored Configuration class.

Reply:
https://github.com/mbta/lamp/pull/5/files#r875077704

this change would make this feel a bit more natural to me as well.